transports/sqs/delayed-delivery.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Delayed Delivery
+summary: Describes the native delayed delivery implementation in the SQS transport
+component: Sqs
+reviewed: 2018-03-02
+versions: '[4,]'
+---
+
+In Versions 4.0 and above, the SQS transport allows [delayed delivery](/nservicebus/messaging/delayed-delivery.md) of messages longer than 15 minutes. The transport creates a FIFO queue per endpoint that allows delaying messages for longer periods of time.
+
+## Enable unrestricted delayed delivery
+
+The unrestricted delayed delivery has to be enabled on the transport configuration:
+
+snippet: DelayedDelivery
+
+Unrestricted delayed delivery needs to be enabled both on the sender and receiver to be able to process delayed deliveries longer than 15 minutes (900 seconds). Delayed delivery below 15 minutes is always supported. The following table illustrates that.
+
+| Scenario                    | Sender   | Receiver | Remarks       |
```

Comment:
`Remark` column with "supported/not supported" is not clear.
If intent is to say "supported/not supported", the column should say so.